libs/@guardian/eslint-plugin-source-foundations/package.json

```diff
@@ -1,27 +1,27 @@
 {
 	"name": "@guardian/eslint-plugin-source-foundations",
-	"version": "10.0.0",
+	"version": "12.0.0",
```

Comment:
I'm a little confused around why this version is going up to `12` but in the CHANGELOG it's listed as `11`: https://github.com/guardian/csnx/pull/469/files#diff-6978ff684633f5f524312cca194f945aec638bf1ff7d2d74b838438da1f3bd47R3

Reply:
Good catch, should be 11.0.0